### DISCUSSION TITLE
ALS Server character stops rotating to control rotation after some minutes

### DISCUSSION THREAD

**Discussion by ToffelsKater** [Sixze/ALS-Refactored#431] [Q&A]
Hey i have an issue when i play in multiplayer the server character stops rotating to the control rotation after a while for clients,
i dont know how to combat this issue, any ideas?

**Comment by ameaninglessname** [comment]
Are you using listen server mode?

**Reply by ToffelsKater** [reply]
Its Client/Server yes

**Comment by ToffelsKater** [comment]
Any news?
